### ISSUE TITLE
deis keys:add should not throw a hard error

### ISSUE THREAD

**Issue by jchauncey** [deis/workflow-cli#91]
```
╰─± deis keys:add
Found the following SSH public keys:
1) deis.pub deis
2) id_boot2docker.pub jonathanchauncey@ENG000637.local
3) id_rsa.pub jonathanchauncey@ENG000637.local
0) Enter path to pubfile (or use keys:add <key_path>)
Which would you like to use with Deis? 4
panic: runtime error: index out of range

goroutine 1 [running]:
github.com/deis/workflow-cli/cmd.chooseKey(0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	/Users/jonathanchauncey/go/src/github.com/deis/workflow-cli/cmd/keys.go:130 +0xc43
github.com/deis/workflow-cli/cmd.KeyAdd(0x0, 0x0, 0x0, 0x0)
	/Users/jonathanchauncey/go/src/github.com/deis/workflow-cli/cmd/keys.go:71 +0xc6
github.com/deis/workflow-cli/parser.keyAdd(0xc820088010, 0x1, 0x1, 0x0, 0x0)
	/Users/jonathanchauncey/go/src/github.com/deis/workflow-cli/parser/keys.go:87 +0x1d7
github.com/deis/workflow-cli/parser.Keys(0xc820088010, 0x1, 0x1, 0x0, 0x0)
	/Users/jonathanchauncey/go/src/github.com/deis/workflow-cli/parser/keys.go:24 +0xbc
main.Command(0xc820088010, 0x1, 0x1, 0x18e0a5)
	/Users/jonathanchauncey/go/src/github.com/deis/workflow-cli/deis.go:105 +0x108a
main.main()
	/Users/jonathanchauncey/go/src/github.com/deis/workflow-cli/deis.go:17 +0x60

goroutine 17 [syscall, locked to thread]:
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:1696 +0x1
```

**Comment by Joshua-Anderson** [comment]
Good catch... I'll fix this up quickly.

**Comment by Joshua-Anderson** [comment]
Fixed in PR #93
